src/GitHubCreateMergePRs/config.xml

```diff
@@ -11,11 +11,11 @@
 
     <!-- Roslyn non-vs-deps branches (flowing between releases) -->
     <merge from="release/dev16.10" to="main" />
-    <merge from="main" to="release/dev17.0" />
+    <!--merge from="main" to="release/dev17.0" /-->
```

Comment:
nit: prefer surrounding the tags a la  `<!-- <merge ... /> -->` instead of turning the tags into comments